app/utils/images.js

```diff
@@ -34,6 +34,9 @@ export const calculateDimensions = (height, width, viewPortWidth = 0, viewPortHe
     ) {
         imageHeight = IMAGE_MIN_DIMENSION;
         imageWidth = imageHeight * heightRatio;
+    } else if (viewPortHeight && imageHeight > viewPortHeight) {
```

Comment:
Can we add test case to cover this?

Reply:
sure

Reply:
this is already covered by the test cases

Reply:
added